### ISSUE TITLE
`goat` polish

### ISSUE THREAD

**Issue by bnewbold** [bluesky-social/indigo#719]
packaging:

- [ ] homebrew cask (?)
- [ ] debian/ubuntu packages
- [ ] command to check for updates
- [ ] maybe move to a separate repo

features:

- [ ] "debug" helpers: custom domain handles, self-hosted ozone, self-hosted PDS
- [ ] lexicon validation
- [ ] generic XRPC endpoint calls
- [ ] PLC validation and create/submit ops
- [ ] account migration
- [ ] ozone: reporting, history, event stream?
- [ ] dump label stream
- [ ] PDS admin: takedown, create accounts, invites

bsky features:

- [ ] parsing text facets
- [ ] post embeds: images, records, URLs
- [ ] bsky.app URL / AT-URI helper (including `goat bsky get`?)
- [ ] timeline, feeds, rendering
- [ ] search
- [ ] some things only work with record types known to indigo; should be generic

interface polish:
- [ ] `--json` output for commands which aren't
- [ ] color: errors, JSON pretty-print
- [ ] allow `-` for stdin

**Comment by tom-sherman** [comment]
Any chance of being able to accept inputs via stdin and/or positional argument as well as the file for `record create`?

**Comment by bnewbold** [comment]
@tom-sherman yes! I'd probably do it by special-casing `-` as the file path, and have folks do `./-` if they really want a file named `-`.

```shell
echo "{whatever}" | jq . | goat record create -
```

Would that work for you?

**Comment by tom-sherman** [comment]
> Would that work for you?

Yes!

**Comment by philion** [comment]
Looking over the polish list, there seems a lot of good, well-defined ways to pitch in (and learn more about ATproto in the process). If I wanted to go do the homebrew cask, do you want a separate issue for that specifically?
